src/modules/AutoinstSoftware.rb

```diff
@@ -985,12 +984,14 @@ def locked_packages
 
     def install_packages
       # user selected packages which have not been already installed
+      # rubocop:disable Lint/UselessAssignment
       packages = Pkg.FilterPackages(
```

Comment:
Why are we keeping this assignment?

Reply:
it is for readability reasons, as it is more obvious what each argument means when you have method with many boolean options. Do you have another idea how to do it?